server/.env.example

```diff
@@ -145,3 +145,13 @@ PA_API_KEY_PRIMARY=MyLongKey
 PA_API_KEY_PRIMARY_TEST=MyLongKey
 PA_API_KEY_SECONDARY=MyLongKeyBackup
 PA_API_KEY_SECONDARY_TEST=MyLongKeyBackup
+
+# The client ID of PA (for use in brave-alert-lib)
+# Get this from Google Cloud --> Brave PA Sign-In --> APIs & Services --> Credentials --> Web ID / Web ID (dev)
+PA_CLIENT_ID=fakeclientid.apps.googleusercontent.com
```

Comment:
would this need to be replaced with a real client ID before deployment? or Google API console would automatically provide a unique client ID ?

Reply:
Hi Ahmad, this environment variable needs to be set before deployment. The line `Get this from Google Cloud --> Brave PA Sign-In --> APIs & Services --> Credentials --> Web ID / Web ID (dev)` shows where you can find the PA client ID in the Google cloud console.

Reply:
Okay. Sound good. Thanks for the clarification.